MaterialLibrary/src/main/java/io/doist/material/drawable/TintDrawable.java

```diff
@@ -83,6 +83,11 @@ public boolean isStateful() {
     }
 
     @Override
+    public boolean setState(int[] stateSet) {
+        return super.setState(stateSet) | super.superSetState(getState());
```

Comment:
If I understand this correctly, we're forwarding the wrapped drawable's states to this drawable to trigger `onStateChange` to update the tint. The `superSetState` method feels like a hack, though. Maybe we could change `WrapperDrawable.setState` to something like this (resembling `Drawable.setState`):
```
public boolean setState(int[] stateSet) {
    if (!Arrays.equals(getState(), stateSet)) {
        boolean stateChanged = mWrapperState.mDrawable.setState(stateSet);
        return onStateChange(stateSet) | stateChanged;
    }
    return false;
}
```
And just remove `TintDrawable.setState` and `WrapperDrawable.superSetState`.

There's an unnecessary semicolon on L260. :)